src/profile-logic/profile-data.js

```diff
@@ -1787,18 +1787,25 @@ export function convertStackToCallNodePath(
  * If no samples are found, 0 is returned.
  */
 export function computeCallNodeMaxDepth(
-  thread: Thread,
+  samples: SamplesLikeTable,
```

Comment:
This is also a fix for the dhat PR. The call node max depth needs the current samples-like table, NOT the thread to compute this correctly.